app/src/main/java/com/example/frogweather/data/City.kt

```diff
@@ -0,0 +1,15 @@
+package com.example.frogweather.data
+
+import com.squareup.moshi.Json
+
+data class City(
+    val id: Int = 0,
+    val name: String = "",
+    @Json(name = "coord")
```

Comment:
Please add @Json for all attributes, due to obfuscation, we can discuss later

Reply:
done